README-V3-UPGRADE.md

```diff
@@ -21,6 +33,14 @@ The packages in `org.spdx.licenseTemplates` are now in the `java-spdx-core` repo
 
 A new class `LicenseTextHelper` was added and the method `isLicenseTextEquivalent(String, String)` along with many supporting methods were moved to `LicenseTextHelper` from `org.spdx.utility.compare.LicenseCompareHelper`.
 
+## Other moved classes
+
+- `org.spdx.library.model.license.ListedLicenses` moved to `org.spdx.library.ListedLicenses`
+
+## Removed methods
+
+Some methods that previously existed in the model classes (e.g. `ListedLicense.getLicenseTextHtml()`) have been removed and do not have an equivalent in v2 of the library.
```

Comment:
@goneall please confirm that this is true. I couldn't find where this method went, but without JavaDoc for the generated model classes I can't be 100% sure it doesn't in fact exist somewhere.